src/cocaine/burlak/config.py

```diff
@@ -431,16 +433,9 @@ def config(self):  # pragma nocover
     @property
     def secure(self):
         secure_conf = self._config.get('secure', {})
+        secure_conf.setdefault('token_expiration_s', Defaults.TOK_UPDATE_SEC)
```

Comment:
Logically it is _view only properties_ and in general case shouldn't change internal config state.